packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -182,12 +182,12 @@ class SimDevice {
   SimDevice(this.category, this.data);
 
   final String category;
-  final Map<String, String> data;
+  final Map<String, dynamic> data;
 
-  String get state => data['state'];
-  String get availability => data['availability'];
-  String get name => data['name'];
-  String get udid => data['udid'];
+  String get state => '${data['state']}';
```

Comment:
I think we can avoid converting to the toString() values here and below, and assume that the values in the map are strings.

Reply:
Done.